### PR TITLE
[test][util] Add 'ExpectClusterQueuesToBeActive'

### PR DESCRIPTION
**Pull request by trasc**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
Add `ExpectClusterQueuesToBeActive` in `test/util` and use it in the integration tests.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
none
```

### CODE REVIEW

File: test/util/util.go [hunk 1]
```diff
@@ -736,3 +736,17 @@ func NewTestingLogger(writer io.Writer, level int) logr.Logger {
...
 	time.Sleep(time.Until(obj.GetCreationTimestamp().Add(time.Second)))
 }
+
+func ExpectClusterQueuesToBeActive(ctx context.Context, c client.Client, cqs ...*kueue.ClusterQueue) {
+	gomega.EventuallyWithOffset(1, func(g gomega.Gomega) {
+		readCq := &kueue.ClusterQueue{}
+		var inactive []string
+		for _, cq := range cqs {
+			err := c.Get(ctx, client.ObjectKeyFromObject(cq), readCq)
+			if err != nil || !apimeta.IsStatusConditionTrue(readCq.Status.Conditions, kueue.ClusterQueueActive) {
```
Comment (tenzen-y) [inline test/util/util.go:746]:
```suggestion
			g.Expect(err).Should(gomega.Succeeded())
			if !apimeta.IsStatusConditionTrue(readCq.Status.Conditions, kueue.ClusterQueueActive) {
```

Shouldn't we verify if the requested CQ exists in advance?

Comment (trasc) [reply test/util/util.go]: I'ts more a question about what we want the result to be:
1. a full list of cqs for which we cannot say their are active (have condition status !True, or we ware not able to get) 
2. the cause of the fist failure

I've changed it to go with 2.

**Pull request merged**